datacatalog/provider.tf

```diff
@@ -0,0 +1,11 @@
+provider "google" {
+  alias = "target"
+}
+
+provider "google" {
+  alias = "dns"
+}
+
+provider "google-beta" {
+  alias = "target"
+}
```

Comment:
Do we need this?

Reply:
Oh good catch! It looks like we're not using this provider anywhere